modules/acquia_cms_site_studio/acquia_cms_site_studio.module

```diff
@@ -220,3 +247,21 @@ function acquia_cms_site_studio_css_alter(array &$css) {
     unset($css[$path]);
   }
 }
+
+/**
+ * Function to add default settings of node revision delete.
+ */
+function _acquia_cms_site_studio_default_node_revision_delete_settings() {
```

Comment:
@apathak18 No procedural code. Move it to separate class.
cc: @chandan-singh7929  / @rajeshreeputra